enterprise/admin/views.py

```diff
@@ -508,16 +509,13 @@ def notify_enrolled_learners(cls, enterprise_customer, request, course_id, users
                 ).format(course_id)
             )
             return
-        course_url = course_details.get('marketing_url')
-        if course_url is None:
-            # If we didn't get a useful path to the course on a marketing site from the catalog API,
-            # then we should build a path to the course on the LMS directly.
-            course_url = get_reversed_url_by_site(
-                request,
-                enterprise_customer.site,
-                'about_course',
-                args=(course_id,),
-            )
+        course_path = urlquote("/courses/{course_id}/info".format(course_id=course_id))
+        lms_root_url = get_configuration_value_for_site(enterprise_customer.site, 'LMS_ROOT_URL')
+        destination_url = "{site}/{login_or_register}?next={course_path}".format(
```

Comment:
Nit: Keep quoting consistent, single vs double. Most of this file  uses single quotes.